lib/js/jsbigints.nim

```diff
@@ -0,0 +1,132 @@
+## Arbitrary precision integers.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+type JsBigInt* = ref object of JsRoot ## Arbitrary precision integer for JavaScript target.
+
+func newBigInt*(integer: cint): JsBigInt {.importjs: "BigInt(#)".}
+  ## Constructor for `JsBigInt`.
+
+func newBigInt*(integer: cstring): JsBigInt {.importjs: "BigInt(#)".}
+  ## Constructor for `JsBigInt`.
+
+func toLocaleString*(this: JsBigInt): cstring {.importjs: "#.$1()".}
+  ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString
+
+func toLocaleString*(this: JsBigInt; locales: cstring): cstring {.importjs: "#.$1(#)".}
+  ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString
+
+func toLocaleString*(this: JsBigInt; locales: openArray[cstring]): cstring {.importjs: "#.$1(#)".}
+  ## https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/toLocaleString
+
+func toString*(this: JsBigInt; radix: cint): cstring {.importjs: "#.$1(#)".}
```

Comment:
cint is only used 4 times in lib/js, and `int` is used plenty of times; of course they all map to Number in js but still, what's your rationale for `cint` here?

Reply:
```console
$ cat example.nim 
echo int.high
echo cint.high
echo int.low
echo cint.low

$ nim r example.nim 
9223372036854775807
2147483647
-9223372036854775808
-2147483648

$ nim js -r example.nim 
2147483647
2147483647
-2147483648
-2147483648

$
```

:shrug: